frontend/src/pages/ChatPage.vue

```diff
@@ -39,6 +39,10 @@
       sendMessage() {
         console.log(this.postMessage);
         this.socket.send(this.postMessage);
+	this.clearMessage();
```

Comment:
What happens when this.socket.send() failed?

Reply:
The postMessage will cleared. I'll try to keep message when the `this.socket.send()` failed. thanks 😄